docs/src/getting-started/wallets.md

```diff
@@ -9,24 +9,22 @@ Wallets are used for many important things, for instance:
 
 The SDK gives you many different ways to create wallets. Let's explore these different ways.
 
-## Creating a wallet from a Private Key
+## Creating a wallet
 
-You can use `fuel_crypto` to generate a random `SecretKey` and use it to create a wallet from it:
+A new wallet can be created from `Option<Provider>`.
 
 ```Rust
-use fuel_crypto::{SecretKey};
-
-// Generate a random private/secret key.
-let mut rng = rand::thread_rng();
-let secret = SecretKey::random(&mut rng);
-
 // Use the test helper to setup a test provider.
 let (provider, _address) = setup_test_provider(vec![]).await;
 
 // Create the wallet.
-let wallet = LocalWallet::new_from_private_key(secret, provider);
+let wallet = LocalWallet::new_from_private_key(Some(provider));
```

Comment:
Reading `new_from_private_key` I would expect to provide my private key, not a `Provider` (which is basically a node)